examples/suit_update/tests/01-run.py

```diff
@@ -77,15 +77,15 @@ def get_ipv6_addr(child):
     if USE_ETHOS == 0:
         # Get device global address
         child.expect(
-            r"inet6 addr: (?P<gladdr>[0-9a-fA-F:]+:[A-Fa-f:0-9]+)"
+            r"inet6 addr: (?P<gladdr>[0-9a-fA-F:]+:[A-Fa-f:0-9]+)\r\n"
```

Comment:
same here, I don't think there's a newline in this regexp, right?

Reply:
Yep, should Have ran tests locally first. Will do now.